webapp/components/FilterMenu/FilterMenu.spec.js

```diff
@@ -9,45 +9,60 @@ localVue.use(Styleguide)
 describe('FilterMenu.vue', () => {
   let wrapper
   let mocks
+  let propsData
 
   const createWrapper = mountMethod => {
     return mountMethod(FilterMenu, {
+      propsData,
       mocks,
       localVue,
     })
   }
 
   beforeEach(() => {
     mocks = { $t: () => {} }
+    propsData = {}
   })
 
-  describe('mount', () => {
+  describe('given a user', () => {
     beforeEach(() => {
-      wrapper = createWrapper(mount)
+      propsData = {
+        user: {
+          id: '4711',
+        },
+      }
     })
 
-    it('renders a card', () => {
-      expect(wrapper.is('.ds-card')).toBe(true)
-    })
-
-    describe('click "filter-by-followed-authors-only" button', () => {
-      it('emits filterBubble object', () => {
-        wrapper.find({ name: 'filter-by-followed-authors-only' }).trigger('click')
-        expect(wrapper.emitted('changeFilterBubble')).toBeTruthy()
+    describe('mount', () => {
+      beforeEach(() => {
+        wrapper = createWrapper(mount)
       })
 
-      it('toggles filterBubble.author property', () => {
-        wrapper.find({ name: 'filter-by-followed-authors-only' }).trigger('click')
-        expect(wrapper.emitted('changeFilterBubble')[0]).toEqual([{ author: 'following' }])
-        wrapper.find({ name: 'filter-by-followed-authors-only' }).trigger('click')
-        expect(wrapper.emitted('changeFilterBubble')[1]).toEqual([{ author: 'all' }])
+      it('renders a card', () => {
+        expect(wrapper.is('.ds-card')).toBe(true)
       })
 
-      it('makes button primary', () => {
-        wrapper.find({ name: 'filter-by-followed-authors-only' }).trigger('click')
-        expect(
-          wrapper.find({ name: 'filter-by-followed-authors-only' }).classes('ds-button-primary'),
-        ).toBe(true)
+      describe('click "filter-by-followed-authors-only" button', () => {
+        it('emits filterBubble object', () => {
+          wrapper.find({ name: 'filter-by-followed-authors-only' }).trigger('click')
+          expect(wrapper.emitted('changeFilterBubble')).toBeTruthy()
```

Comment:
At the risk of you getting annoyed, and not saying I wouldn't be happy to merge as is, I am not a fan of these tests that check if something is `toBeTruthy()`. I feel like these are flaky. 

I am aware that in the documentation they use them https://vue-test-utils.vuejs.org/api/wrapper/#emitted
so maybe it's something I need to get over, but I personally would write it `expect(wrapper.emitted().changeFilterBubble.length).toBe(1)`, for what it's worth

though I see below that you are testing what the emit event is called with, which I really like :smile_cat:

Reply:
ah, yeah, that test is just to check that it emits anything after all..

Reply:
yes, I got that, and I guess it would never be Truthy unless it actually emits something, which is probably why they have it in the examples in the docs... just makes me a lil' nervous hehe